frontend/taipy-gui/src/components/Taipy/Notification.tsx

```diff
@@ -56,20 +54,19 @@ const TaipyNotification = ({ notifications }: NotificationProps) => {
 
     useEffect(() => {
         if (notification) {
-            const notificationId = notification.notificationId || "";
-            if (notification.atype === "") {
-                closeSnackbar(notificationId);
+            if (notification.nType === "") {
+                closeSnackbar(notification.snackBarId);
             } else {
                 enqueueSnackbar(notification.message, {
-                    variant: notification.atype as VariantType,
+                    variant: notification.nType as VariantType,
                     action: notificationAction,
-                    autoHideDuration: notification.duration,
-                    key: notificationId,
+                    key: notification.snackBarId,
+                    autoHideDuration: (notification.duration > 0 && notification.notificationId) ? notification.duration : null
```

Comment:
If notification id is not defined by the dev, the notification will stay forever??

Reply:
The other way around.
No id -> use duration. If 0, then it shows and goes in a millisecond.

Reply:
Indeed! How could I fail on this one... THX!